config/locales/de.yml

```diff
@@ -1148,44 +1161,44 @@ de:
     order_already_updated: Bestellung bereits geändert
     order_approved: Bestellung freigegeben
     order_canceled: Bestellung storniert
+    order_completed: Bestellung kostenpflichtig abgegeben
```

Comment:
The "kostenpflichtig" is only necessary for frontend users and we do not use this translation in the frontend.

This is only used in the backend flash notice and should better be called "Bestellung erfolgreich abgeschlossen" (so it fits the button on the confirm page)

Reply:
done